/*
 *      Polygraph (release 0.1)
 *      Signature generation algorithms for polymorphic worms
 *
 *      Copyright (c) 2004-2005, Intel Corporation
 *      All Rights Reserved
 *
 *  This software is distributed under the terms of the Eclipse Public
 *  License, Version 1.0 which can be found in the file named LICENSE.
 *  ANY USE, REPRODUCTION OR DISTRIBUTION OF THIS SOFTWARE CONSTITUTES
 *  RECIPIENT'S ACCEPTANCE OF THIS AGREEMENT
 */


--- lst_structs.h.old	2005-06-01 16:05:27.000000000 -0400
+++ lst_structs.h	2005-06-01 16:05:35.000000000 -0400
@@ -69,6 +69,7 @@
 
   u_int32_t                   visitors;
   u_int                       bus_visited;
+  void                       *annotation;
 };
 
 
